### PR TITLE
fix json api

### PR DESCRIPTION
**Pull request by mmaxim**
@patrickxb @songgao

### CODE REVIEW

File: go/client/cmd_chat_api.go [hunk 1]
```diff
@@ -311,7 +311,6 @@ func (c *CmdChatAPI) ReadV1(ctx context.Context, opts readOptionsV1) Reply {
 			return c.errReply(libkb.NewChatMessageVersionError(version))
 		}
 
-		return c.errReply(errors.New("unexpected response from service: UnboxingError and Message are both empty"))
```
Comment (songgao) [inline go/client/cmd_chat_api.go]: It looks like this should be there, but the issue is we are missing a `continue` after line 309?

Comment (mmaxim) [inline go/client/cmd_chat_api.go]: How would I get to this line if there was a continue on 309?

Comment (songgao) [inline go/client/cmd_chat_api.go]: Oh no sorry. There should be a `if m.Message != nil {` before line 277 and a `}` after line 312 as well

Comment (mmaxim) [inline go/client/cmd_chat_api.go]: Ok I'll make that change.

Comment (songgao) [inline go/client/cmd_chat_api.go]: Sounds good; Thanks Mike

**Pull request merged**